cvat-core/src/api-implementation.ts

```diff
@@ -390,19 +405,56 @@ export default function implementAPI(cvat) {
             };
         }
 
-        const reportsData = await serverProxy.analytics.quality.conflicts(updatedParams);
+        const conflictsData = await serverProxy.analytics.quality.conflicts(updatedParams);
+        const conflicts = conflictsData.map((conflict) => new QualityConflict({ ...conflict }));
+        conflicts.count = conflictsData.count;
+        return conflicts;
+    };
 
-        return reportsData.map((conflict) => new QualityConflict({ ...conflict }));
+    cvat.analytics.quality.settings.get.implementation = async (filter: any) => {
+        interface FilterParams {
+            id?: number, taskId?: number, projectId?: number
+        }
+        const { id, taskId, projectId }: FilterParams = filter;
+
+        const settings = await serverProxy.analytics.quality.settings.get(id, taskId, projectId);
+        if (settings) {
+            return new QualitySettings({ ...settings });
+        }
+        return null;
     };
 
-    cvat.analytics.quality.settings.get.implementation = async (taskID: number) => {
-        const settings = await serverProxy.analytics.quality.settings.get(taskID);
-        return new QualitySettings({ ...settings });
+    cvat.analytics.quality.settings.update.implementation = async (settingsId: number, values: any) => {
```

Comment:
According to current approach, update should be a method of QualitySettings class

Reply:
Removed